core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -1525,7 +1525,7 @@ public String toString() {
           .build();
   public static final PropertyKey MASTER_WORKER_HEARTBEAT_INTERVAL =
       new Builder(Name.MASTER_WORKER_HEARTBEAT_INTERVAL)
-          .setDefaultValue("10sec")
+          .setDefaultValue("1sec")
```

Comment:
@calvinjia This was change to 10s a while ago, what was the reason for that, and do we want to change it back to 1s with this PR?

Reply:
I think it was in an attempt to change the worker heartbeat, but it only affects the lost file detector. I don't think we need to run the lost file detector so frequently, perhaps we should rename this key and keep 10 seconds.

Reply:
Previously our worker heartbeat takes a long time because of the sync metrics processing, so we update the `WORKER_BLOCK_HEARTBEAT_INTERVAL` which controls the interval of workerS calling worker heartbeat to 10s.

To be efficient, `MASTER_WORKER_HEARTBEAT_INTERVAL` should be similar to the value of `WORKER_BLOCK_HEARTBEAT_INTERVAL`. For example, `MASTER_WOREKR_HEARTBEAT_INTERVAL` controls the interval of finding lost workers and lost files which are updated by workerHeartbeat (controlled by `WORKER_BLOCK_HEARTBEAT_INTERVAL`)

Reply:
@calvinjia It also affects the lost worker detector, this part may be more critical

Reply:
Oh right, though our worker timeout is 5 minutes so I think plus up to 10 seconds of additional delay is fine. Renaming it is mainly to avoid the confusion that it is the same as the actual heartbeat interval coming from the worker.